src/Mvc/test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/ValidationSummaryTagHelperTest.cs

```diff
@@ -619,6 +679,27 @@ private static void SetValidModelState(ModelStateDictionary modelState)
             }
         }
 
+        public static TheoryData<ModelStateDictionary, string> ProcessAsync_GeneratesExpectedOutput_WithErrorsData
```

Comment:
Please move this method just above `ProcessAsync_GeneratesExpectedOutput_WithModelErrorForIEnumerable(...)`. (We usually only have support methods and private classes at the bottom of test classes.)